src/yardstick/tool/grype.py

```diff
@@ -154,9 +154,34 @@ def _install_from_git(
 
         return Grype(path=path, version_detail=description, **kwargs)
 
+    @classmethod
+    def _install_from_path(
+        cls,
+        path: Optional[str],
+        src_path: str,
+        **kwargs,
+    ) -> "Grype":
+        # get the description and head ref from the repo
+        src_repo_path = os.path.abspath(src_path)
```

Comment:
might need an `expanduser` call here similar to https://github.com/anchore/vunnel/blob/ee45d4e3e2c69adf609c65ebd94fda21169d41a5/tests/quality/configure.py#L452